AppleALC/AppleALC-Info.plist

```diff
@@ -62,17 +62,15 @@
 		<key>com.apple.iokit.IOPCIFamily</key>
 		<string>1.0.0b1</string>
 		<key>com.apple.kpi.bsd</key>
-		<string>10.0.0</string>
-		<key>com.apple.kpi.dsep</key>
```

Comment:
is dsep really optional? Can you check with kextlib utility? It is built into macOS.

Reply:
dsep is not present in 10.4, and is not directly used within this plugin.